lib/fs/util.go

```diff
@@ -48,32 +48,44 @@ func getHomeDir() (string, error) {
 	return os.UserHomeDir()
 }
 
-const windowsDisallowedCharacters = (`<>:"|?*` +
+// On Android, the following characters are disallowed on user storage
+// even if the filesystem is not FAT. On Windows, they apply to both FAT
+// and NTFS.
+// Ref: https://cs.android.com/android/platform/superproject/main/+/main:packages/providers/MediaProvider/src/com/android/providers/media/util/FileUtils.java;drc=9d054a9eb6a8d8c3d79e007ea6d2f89c94a52f06;l=485
+const fatDisallowedCharacters = (`/\<>:"|?*` +
```

Comment:
androidDisallowedCharacters?

Reply:
Isn't `fat` better in this case? It's more generic and that's essentially where these characters come from (even though they are applied on a wider scale than just on `FAT`).

Reply:
I can imagine someone reading the code later on thinking "what the hell, you can't blindly assume that the user has a fat filesystem. \*reads comment\* Ah, that's why".